whotracksme/data/assets/trackerdb.sql

```diff
@@ -555,6 +555,24 @@ INSERT INTO "companies" VALUES('samba_tv','Samba TV',NULL,'https://samba.tv/lega
 INSERT INTO "companies" VALUES('sape','Sape Sapient Solution',NULL,NULL,'https://www.sape.ru/en',NULL);
 INSERT INTO "companies" VALUES('schibsted_asa','Schibsted ASA',NULL,NULL,'http://www.schibsted.com/',NULL);
 INSERT INTO "companies" VALUES('sentifi','Sentifi AG',NULL,'https://sentifi.com/page/privacy/','https://sentifi.com/',NULL);
+INSERT INTO "companies" VALUES('orange_france','Orange France',NULL,'http://c.orange.fr/donnees-personnelles.html','https://www.orange.fr/',NULL);
+INSERT INTO "companies" VALUES('dailymotion','Dailymotion',NULL,'http://www.dailymotion.com/legal/privacy','http://www.dailymotion.com/',NULL);
```

Comment:
This entry is no longer used (dailymotion is owned by Vivendi).

Reply:
Dailymotion deleted